src/components/v5/common/ActionSidebar/partials/forms/EditColonyDetailsForm/hooks.ts

```diff
@@ -61,12 +61,10 @@ export const useEditColonyDetails = (
             motionDomainId: payload.createdIn,
             decisionMethod: payload.decisionMethod,
             annotationMessage: payload.description,
-            externalLinks: payload.externalLinks.map(
-              ({ linkType, ...link }) => ({
-                ...link,
-                name: linkType,
-              }),
-            ),
+            externalLinks: payload.externalLinks.map(({ name, link }) => ({
+              link,
+              name,
+            })),
```

Comment:
Do we actually need that map?

Reply:
huh good point, I don't really think we do :D

Reply:
You only might get extra properties but that shouldn't matter